src/mem/cache/prefetch/pif.cc

```diff
@@ -176,14 +176,16 @@ PIF::notifyRetiredInst(const Addr pc)
                 // Insert the spatial entry into the history buffer and update
                 // the 'iterator' table to point to the new entry
                 historyBuffer.push_back(spatialCompactor);
-
-                auto idx_entry = index.findEntry(spatialCompactor.trigger);
+                bool is_secure = false;
```

Comment:
Since the `is_secure` variable is not changed afterwards, can you change this to `constexpr bool is_secure = false;`?